packages/algob/src/internal/core/tasks/builtin-tasks.ts

```diff
@@ -13,3 +13,10 @@ export default function (): void{
     loadPluginFile(path.join(basedir, t[1]));
   }
 }
+
+// checks if the task name is not a setup kind of task
+export function isSetupTask (taskName: string): boolean {
+  return taskName === tasks.TASK_HELP ||
+    taskName === tasks.TASK_CLEAN ||
```

Comment:
Also clean shouldn't be allowed to be run outside of a project so it's not a setup task

Reply:
Yes. I will remove it.

Reply:
thanks for catching this